akka-projection-cassandra/src/test/java/akka/projection/cassandra/CassandraProjectionTest.java

```diff
@@ -82,12 +81,12 @@ public static void afterAll() throws Exception {
     TestSourceProvider(String entityId) {
       this.entityId = entityId;
       envelopes = Arrays.asList(
-        new Envelope(entityId, 1, "abc"),
-        new Envelope(entityId, 2, "def"),
-        new Envelope(entityId, 3, "ghi"),
-        new Envelope(entityId, 4, "jkl"),
-        new Envelope(entityId, 5, "mno"),
-        new Envelope(entityId, 6, "pqr"));
+              new Envelope(entityId, 1, "abc"),
+              new Envelope(entityId, 2, "def"),
+              new Envelope(entityId, 3, "ghi"),
+              new Envelope(entityId, 4, "jkl"),
+              new Envelope(entityId, 5, "mno"),
+              new Envelope(entityId, 6, "pqr"));
```

Comment:
I guess we haven't added java formatting to the build yet, but please adjust intellij settings to use two spaces

Reply:
Thanks.  Done.